### PR TITLE
Doc edits

### LINKED ISSUE #15: Proofread Camp Docs 📝
This is a [starter issue](http://hood.ie/blog/starter-issues.html). It’s reserved for participants of [OSCON Contribute](https://ti.to/hoodie/oscon-meet-the-hoodies) until May 20th, 2016. 
### :octocat:  Your First Pull Request?

We would like to take the opportunity to invite someone who did not yet contribute to Hoodie to work on this issue. Follow [@YourFirstPR](https://twitter.com/yourfirstPR) on Twitter for more of these issues across many open source projects :)
### 🤔 What you will need to know

Strong command of English language/grammar. Knowledge of code (JavaScript, Node.js) is not essential, but helpful. 
### 🐞 The Issue

Proofread and suggest corrections to the new Camp documentation found here: http://docs.hood.ie/camp/.
### :dart: The goal

Finding typos, formatting errors, etc. – any small error that might distract a new contributor when trying to learn from the new documentation.

Helpful resource: [our glossary of commonly used terms](https://github.com/hoodiehq/editorial/wiki/Glossary-of-common-terms). Double check your references to popular tech lingo here. 
### :clipboard: Step by Step
- [x] **claim this issue**  
  comment below. Once claimed we add you as contributor to this repository, so we can assign you to the ticket and you can start checking off the checkboxes below as you progress. We will also replace the `ready` label with `in progress`.
- [x] **Review** the documentation beginning on the [welcoming page](http://docs.hood.ie/camp/.) all the way through the [glossary](http://docs.hood.ie/camp/hoodieverse/glossary.html). 
- [x] **Start a Pull Request**. hoodiehq/documentation#219 
- [x] **Done** :+1: Ask in comments for a review :)

Ping us in the [Hoodie Chat](http://hood.ie/chat/) or on [Twitter](https://twitter.com/hoodiehq/) if you have any questions :)

### PR DESCRIPTION
**Pull request by rossyoung5**
closes https://github.com/hoodiehq/camp/issues/15

I made several edits to the documentation for Hoodie, mostly typos and grammar. If you have any questions about the changes, just let me know.

### CODE REVIEW

File: camp/hoodieverse/hoodie-concepts.md [hunk 1]
```diff
@@ -41,17 +43,17 @@ With Hoodie, you scaffold out your app with
...
 " target="_blank">Offline First</a>.
 
-**Offline First means: build your apps without the assumption of permanent connectivity**. Cache data and apps locally. Build interfaces that accomodate the offline state elegantly. Design user interactions that will not break if their train goes into a tunnel. Don't freak out your users with network error messages or frustrate them with inaccessible data. **Offline First apps are faster, more robust, more pleasant to use, and ultimately: more useful.**
+**Offline First means: build your apps without the assumption of permanent connectivity**. Cache data and apps locally. Build interfaces that accomodate the offline state elegantly. Design user interactions that will not break if their train goes into a tunnel. Don't freak out your users with network error messages or frustrate them with inaccessible data. **Offline First apps are faster, more robust, more pleasant to use, and ultimately, more useful.**
```
Comment (jsoranno) [inline camp/hoodieverse/hoodie-concepts.md]: Nice catch, rossyoung5.

File: camp/hoodieverse/hoodie-concepts.md [hunk 2]
```diff
@@ -30,7 +30,9 @@ Here's some <a href="http://nobackend.org/dreamcode.html" target="_blank">furthe
...
 
-People have been building web apps for quite a while now, and their basic operations (sign up, sign in, sign out, store and retrieve data, etc.) must have been written a million separate times by now. These things really shouldn't be difficult anymore. So we're proposing Hoodie as a noBackend solution. Yes, a backend does exist, but it doesn't have to exist in your head. You don't have to plan it or set it up. You simply don't have to worry about it for those basic operations, you can do all of them with Hoodie's frontend API. Of yourse, we let you dig as deep as you want, but for the start, you don't have to.
+People have been building web apps for quite a while now, and their basic operations (sign up, sign in, sign out, store and retrieve data, etc.) must have been written a million separate times by now. These things really shouldn't be difficult anymore. So we're proposing Hoodie as a noBackend solution.
```
Comment (jsoranno) [inline camp/hoodieverse/hoodie-concepts.md:33]: I like how it's broken out into two paragraphs.

File: camp/techdocs/concepts/worker.md [hunk 3]
```diff
@@ -572,7 +574,7 @@ And then we replace all occurrences in our code, as shown in this diff:
 
 Now that our code is variable, we need a way to pass in new configuration options.
 
-This is easy, we just extend what we already have:
+This is easy; we just extend what we already have:
 
     var config = {
```
Comment (jsoranno) [inline camp/techdocs/concepts/worker.md:579]: Semicolons are the best.

File: camp/hoodieverse/hoodie-concepts.md (hunk 2 above)
Comment (gr2m) [inline camp/hoodieverse/hoodie-concepts.md:33]: Agree 👍

**Comment by gr2m** [comment]
Fantastic work, thanks so much @rossyoung5 👏 👍 🙇 

And welcome at Hoodie :) Your help is much appreciated, docs is exactly what we need the most help with right now :) I’ve invited you to the Hoodie Contributors Team! Are you on twitter? We like giving shout outs to new contributors

**Comment by rossyoung5** [comment]
Thanks! Glad to help and would love to contribute more. I'm @MRossYoung on
Twitter.

Cheers,
Ross

**Pull request merged**